### ISSUE TITLE
Help text problem

### ISSUE THREAD

**Issue by russel** [gradle/gradle#3344]
```
|> gradle -version
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.codehaus.groovy.reflection.CachedClass (file:/home/users/russel/.sdkman/candidates/gradle/4.3/lib/groovy-all-2.4.12.jar) to method java.lang.Object.finalize()
WARNING: Please consider reporting this to the maintainers of org.codehaus.groovy.reflection.CachedClass
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release

------------------------------------------------------------
Gradle 4.3
------------------------------------------------------------

Build time:   2017-10-30 15:43:29 UTC
Revision:     c684c202534c4138b51033b52d871939b8d38d72

Groovy:       2.4.12
Ant:          Apache Ant(TM) version 1.9.6 compiled on June 29 2015
JVM:          9.0.0.15 (Azul Systems, Inc. 9.0.0.15+181)
OS:           Linux 4.13.0-1-amd64 amd64
```

There are two options from the `gradle --help` output that are particularly interesting:
```
--daemon                  Uses the Gradle Daemon to run the build. Starts the Daemon if not running.
```
```
--no-daemon               Uses the Gradle Daemon to run the build. Starts the Daemon if not running.
```

**Comment by oehme** [comment]
Duplicate of #3237

**Comment by russel** [comment]
One can also add:
```
--parallel                Build projects in parallel. Gradle will attempt to determine the optimal number of executor threads to use. [incubating]
```
```
--no-parallel             Build projects in parallel. Gradle will attempt to determine the optimal number of executor threads to use. [incubating]
```

**Comment by russel** [comment]
Not to mention:
```
--scan                    Creates a build scan. Gradle will emit a warning if the build scan plugin has not been applied. (https://gradle.com/build-scans) [incubating]
```
```
--no-scan                 Creates a build scan. Gradle will emit a warning if the build scan plugin has not been applied. (https://gradle.com/build-scans) [incubating]
```

**Comment by bmuschko** [comment]
All those option have been fixed in the PR. Feel free to try out a nightly.

**Comment by wcdeich4** [comment]
If this was fixed in Gradle 4.3, then Gradle 4.3.1 appears to have a reversion. 
>gradle -v
WARNING: An illegal reflective access operation has occured
WARNING: Illegal reflective access b y org.codehaus.groove.reflection.CachedClass (file:/C:/Users/me/Gradle/gradle-4.3.1/lib/groovy-all-2.4.12.jar) to method java.lang.Object.finalize()

>java -version 
java version "9.0.1"
Java(TM) SE Runtime Environment (build 9.0.1+11)
Java HotSpot(TM) 64-Bit Server VM (build 9.0.1+11,  mixed mode)

**Comment by bmuschko** [comment]
@wcdeich4 Looks like I didn't link the [pull request](https://github.com/gradle/gradle/pull/3237) as I meant to do. This has only been fixed with 4.4-RC1 which we'll likely release in ~ 1 1/2 weeks. Is it correct to assume that you are not referring to the WARNING you posted above?

**Comment by wcdeich4** [comment]
Thanks for clarifying.  I'm not sure what you mean by "Is it correct to assume that you are not referring to the WARNING you posted above?" I decided to post a comment b/c it looks like my warning message is basically the same as the one @russel got.

**Comment by bmuschko** [comment]
@wcdeich4 The warning message you see is a known issue originating from Groovy. You can find more information here: https://github.com/gradle/gradle/issues/2995

**Comment by wcdeich4** [comment]
Ok. Then maybe that warning is not related to my real problem.  I'm setting up Eclipse & Gradle on a new PC, but when I create a new Gradle  Project ... & click Finish, nothing happens.  Not sure what I did wrong; I followed the Gradle Install guide & I have the BuildShip 2.0 plugin for Eclipse

**Comment by bmuschko** [comment]
@wcdeich4 Could you please create a new issue that describes your issue?

**Comment by wcdeich4** [comment]
Ok, thanks.  I didn't think you'd want to bother with that

**Comment by bmuschko** [comment]
@wcdeich4 I think your issue has a different root cause than this one. That's why I am asking for a new issue that describes your environment, build script and operations you run. We are happy to help you get started.
